zygrader/data/model.py

```diff
@@ -126,14 +133,17 @@ def from_json(cls, section_json):
         default_due_time = (datetime.datetime.strptime(
             default_due_time_str,
             ClassSection.DUE_TIME_STORAGE_FORMAT).astimezone(tz=None).time())
-        return ClassSection(section_number, default_due_time)
+        # Section Groups were added later, so do a safer get to avoid KeyError
+        section_group = section_json.get("section_group")
```

Comment:
Would you want to add a default here, or is `None` the best default? For example, you could do `default="Default Section Group"` as the second parameter.

Reply:
I think None works well. The constructor handles this case, and I'd rather the special string be in just one location.